ldap/src/main/java/org/springframework/security/ldap/server/EmbeddedDsContainer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.ldap.server;
+
+/**
+ * Common interface for embedded DS containers.
+ *
+ * @author Evgeniy Cheban
+ * @see ApacheDSContainer
+ * @see UnboundIdContainer
+ */
+public interface EmbeddedDsContainer {
```

Comment:
I don't think we should need to add a new public interface.

Reply:
Removed.